### DISCUSSION TITLE
jQuery no longer loading Shakapacker 8.0.2

### DISCUSSION THREAD

**Discussion by RailsCod3rFuture** [shakacode/shakapacker#517] [Q&A] [answered]
I tried loading jquery through expose loader and also loading it directly in the application.js. Doesn't seem like anything is working for it

steps to reproduce
create a rails 7.2.1 app
install shakapacker 8.0.2
yarn add jquery
yarn add expose-loader -D
import $ from "expose-loader?exposes=$,jQuery!jquery";

--- you'll see that jquery doesn't load at all...Causes all js to fail in application.js

**Comment by justin808** [comment, marked as answer]
Check this out: https://github.com/search?q=repo%3Ashakacode%2Freact-webpack-rails-tutorial%20jquery&type=code
